include/fieldmap.h

```diff
@@ -34,7 +34,7 @@ u16* GetVarPointer(FieldSystem* fieldSystem, u16 varId);
 u16 FieldSystem_VarGet(FieldSystem* fieldSystem, u16 varId);
 BOOL FieldSystem_VarSet(FieldSystem* fieldSystem, u16 varId, u16 value);
 u16 FieldSystem_VarGetObjectEventGraphicsId(FieldSystem* fieldSystem, u16 objId);
-BOOL FieldSystem_FlagGet(FieldSystem *fieldSystem, u16 flagId);
+BOOL FieldSystem_FlagCheck(FieldSystem *fieldSystem, u16 flagId);
```

Comment:
verb should come before? I remember having a discussion about this but that might've just been for the battle commands

Reply:
out of scope (will fix it when I get to fieldmap)